src/components/Button/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { openModal } from './../../actions/index';
+
+import './Button.scss';
+
+const Button = ({ type, icon, children, link, modal, click, disabled }) => {
+  let plusIcon = icon ? <span>+ </span> : null;
+  let dispatch = useDispatch();
+
+  if (link) {
+    if (link.charAt(0) === '/') return <Link to={link} className='button' type={type}>{plusIcon}{children}</Link>;
```

Comment:
in prod does this work? i thought relative urls become `https://whateverrootdomain.com/`

Reply:
It should work. The link variable is just the input for the ACF field. 

When you enter the button link input, it should either be `/example` for an internal link or `https://www.example.com` for an external link. 

If there's no `/` in front it will be treated as an `<a>` tag

Reply:
Can we get a more robust solution then (ie what if a user added `http://huoa.org` in acf field)? A nice regex should work.

Reply:
I added a regex that checks for it. If it matches then it converts it into a relative path per the Root URL. Would that work?